### PR TITLE
Include recent changes in RPM spec dependencies.

### PR DESCRIPTION
**Pull request by gordonmessmer**
These changes are required in order for bcfg2 to run if installed from a clean, minimal installation.

### CODE REVIEW

File: misc/bcfg2.spec [hunk 1]
```diff
@@ -137,6 +139,8 @@ BuildRequires:    systemd-units
 Requires:         python-ssl
 %endif
 Requires:         libselinux-python
+Requires:         pylibacl
```
Comment (stpierre) [inline misc/bcfg2.spec]: pylibacl is only used by the client, and it's optional. Bcfg2 should run just fine without it.

**Comment by gordonmessmer** [comment]
The same is true of libselinux-python, so I assumed they'd be treated the same.  Is that the only change you'd like me to back out?

**Comment by stpierre** [comment]
It looks like we're quite inconsistent about what we treat as required/optional in the spec file, and this actually is in the right section, so you can leave it. LGTM as-is.

File: misc/bcfg2.spec [hunk 2]
```diff
@@ -18,8 +18,8 @@
 # characters from the appropriate line below.
 #
 # Don't forget to change the Release: tag below to something like 0.1
-#%%global _rc 1
-%global _pre pre1
+%global _rc 1
+#%%global _pre 1
```
Comment (solj) [inline misc/bcfg2.spec]: Aren't these set properly by the _pre_rc bits below. It looks to me like that's what the build script uses when creating a new release.

Comment (gordonmessmer) [inline misc/bcfg2.spec]: They probably should be commented, yes.  

A couple of things look broken, though.  The "release" tag has a macro that requires a number, not "rc" or "pre" suffixed with a number.   If the export script defines _pre as "pre1", the macro will expand that to "prepre1".  

Seems like a bad idea to modify that file in place, too.  Inevitably, someone will run export to tag an rc or pre release, and then later check che changes in to git, unintentionally.  That's probably why _pre is uncommented now.

**Comment by solj** [comment]
It appears as though your master branch also contains some unrelated jinja changes. Perhaps you want to create a couple topical branches for each of these?

**Comment by gordonmessmer** [comment]
I did previously submit a prior pull request from this master branch, but those were merged.  In this pull request, I only see misc/bcfg2.spec being modified.

**Comment by solj** [comment]
My apologies. My local branch wasn't up to date. Merged, thanks!

**Pull request merged**